app/jobs/zendesk_tickets_job.rb

```diff
@@ -14,16 +14,21 @@ def perform(feedback)
       fail 'Cannot create Zendesk ticket since Zendesk not configured'
     end
 
-    ZendeskAPI::Ticket.create!(
-      Rails.configuration.zendesk_client,
-      ticket_attrs(feedback))
+    feedback.destroy! if ticket_raised?(feedback)
   end
 
 private
 
   # We have 2 Zendesk inboxes configured, one for the public that matches that
   # the service field is 'prison_visits' and another one for staff that matches
   # tickets tagged with 'staff.prison.visits'.
+
+  def ticket_raised?(feedback)
```

Comment:
idiomatically in ruby methods that ends with a `?` returns true or false and have no side effects. I suggest you rename this method to `#raise_ticket!`